### ISSUE TITLE
Bad cycling and walking routing

### ISSUE THREAD

**Issue by cypok** [organicmaps/organicmaps#9419]
Cycling and walking route from 48.195927, 11.52385 to 48.197874, 11.524206 contains the large and small detour:
<img src="https://github.com/user-attachments/assets/c1cee2f8-eb67-45d7-872a-b31defaaef4b" width="40%"/> <img src="https://github.com/user-attachments/assets/24700bab-f455-41f5-a045-fd057485e738" width="40%"/>

Valhalla, GraphHopper and OSRM suggest direct route for both cycling and walking.

App version 2024.09.08-2

**Comment by map-per** [comment]
The problem is that the road section https://www.openstreetmap.org/way/613250476 is tagged with `foot=use_sidepath` and `bicycle=use_sidepath`.

Related to #4762

**Comment by map-per** [comment]
I fixed the OpenStreetMap data (see: https://www.openstreetmap.org/changeset/157369154).

IMO the OSM `use_sidepath` data is not very reliably in many cases, so we should take this tags less serious in the routing algorithm. My suggestion is to just use a "medium high" cost penalty for using such a road instead of not using it at all.
Such a "medium high" penalty should be high enough to favor a close by separate foot- or cycleway. But in case there is no separate way and the `use_sidepath` is a mapping error the algorithm can still take the the road.
(@vng What do you think?)

**Comment by vng** [comment]
@map-per I agree that we should treat it with some kind of "medium penalty", but it is not easy to fix now.

Let's postpone it until better "surface tag -> penalty" mapping (we have open issues). These problems are similar.
